app/views/bets/show.html.erb

```diff
@@ -1,56 +1,77 @@
+  <hr>
+  <div class="row">
+    <div class="col col-lg-6 offset-lg-3">
+      <div class="card">
+        <div class="card-header text-muted">
+          <div class="row">
+            <div class="col col-lg-12 text-center">
+              <h4 class="card-header-text">Participants</h4>
+            </div>
+          </div>
+        </div>
+        <div class="card-block">
+          <table class="table table-hover">
+            <thead class="thead-default">
+              <tr class="row">
+                <th class="col col-lg-4">Username</th>
+                <th class="col col-lg-4">Option #</th>
+                <th class="col col-lg-4">Amount</th>
+              </tr>
+            </thead>
+            <tbody>
+              <%= render @bet.user_bets %>
+            </tbody>
+          </table>
         </div>
-      </div>
-      <div class="card-block">
-        <table class="table table-hover">
-          <%= render @bet.user_bets %>
-        </table>
       </div>
     </div>
   </div>
-</div>
 
-<hr>
-<div class="row">
-  <div class="col col-lg-6 offset-lg-3">
-    <div class="card">
-      <div class="card-header text-muted">
-        <div class="row">
-          <div class="col col-lg-4">Option ID</div>
-          <div class="col col-lg-4">Text</div>
-          <div class="col col-lg-4">Option total</div>
+  <hr>
+  <div class="row">
+    <div class="col col-lg-6 offset-lg-3">
+      <%= form_for [@bet, @user_bet] do |f| %>
+        <div class="card">
+          <div class="card-header text-muted">
+            <div class="row">
+              <div class="col col-lg-12 text-center">
+                <h4 class="card-header-text">Betting Options</h4>
+              </div>
+            </div>
+          </div>
+          <div class="card-block">
+            <table class="table table-hover table-selectable">
+              <thead class="thead-default">
+                <tr class="row">
+                  <th class="col col-lg-2">#</th>
+                  <th class="col col-lg-6">Description</th>
+                  <th class="col col-lg-4">$ total</th>
+                </tr>
+              </thead>
+              <tbody>
+                <% @bet.options.each_with_index do |option, index| %>
+                  <%= render 'bet_options/bet_option_radio', bet_option: option,
+                    f: f, bet_option_counter: index %>
+                <% end %>
+              </tbody>
+            </table>
+          </div>
+          <div class="card-footer">
+            <div class="form-group">
+              <div class="input-group">
+                <span class="input-group-addon"><i class="fa fa-usd"></i></span>
+                <%= f.hidden_field :user_id, value: current_user.id %>
+                <%= f.text_field :amount_bet, class: 'form-control' %>
+              </div>
+            </div>
+            <%= f.submit 'Bet!', class: 'btn btn-success',
+              role: 'button', style: 'width: 100%;', method: 'get' %>
```

Comment:
Method should go on the form element
:-1: inline styles. Styles should always be written in external style sheets